chapter_recurrent-neural-networks/language-models-and-dataset.md

```diff
@@ -53,11 +53,11 @@ P(x_1, x_2, x_3, x_4) &=  P(x_1) P(x_2  \mid  x_1) P(x_3  \mid  x_1, x_2) P(x_4
 \end{aligned}
 $$
 
-涉及一个、两个和三个变量的概率公式通常分别称为“一元语法”（unigram）、“二元语法”（bigram）和“三元语法”（trigram）。下面，我们将学习如何设计更好的模型。
+通常，涉及一个、两个和三个变量的概率公式分别被称为“一元语法”（unigram）、“二元语法”（bigram）和“三元语法”（trigram）模型。下面，我们将学习如何去设计更好的模型。
 
 ## 自然语言统计
 
-让我们看看如何对真实数据起作用。根据 :numref:`sec_text_preprocessing` 中介绍的时光机器数据集构建词汇表，并打印最常用的10个单词。
+让我们看看在真实数据上效果如何。根据 :numref:`sec_text_preprocessing` 中介绍的时光机器数据集构建词汇表，并打印前 $10$ 个最常用的（频率最高的）单词。
```

Comment:
Hey, please don't worry about adding $...$ to the text. The publisher don't want us include any extra space in the text... Please [PR 868](https://github.com/d2l-ai/d2l-zh/pull/868) for more detail.

Reply:
(I know the space problem may exist everywhere...) We will remove the space for all the sections together.

Reply:
If I should remove these space between Chinese characters and English words/special fonts?
I added these spaces in Typora editor for good vision which can help me to find errors.

Reply:
I see. You don't need to worry too much on that, I will figure out a way to remove the spaces for all section later. Thanks!

Reply:
One more question, could you give me a principle about when and where I can use space between these marks(e.g. reference, mathematics equation, code in the main body, etc.)?
Now, I do it based on my visual effect, but I thought it is not a normative way.
Thks.

Reply:
Great question! Just make some notes in style_guide (PR #889). Please let me know if that is clear enough to you?!